tests/torchtune/datasets/test_chat_dataset.py

```diff
@@ -14,18 +14,37 @@
 from torchtune.datasets import ChatDataset
 
 
-class DummyTemplate:
-    def __init__(self):
-        self.template = {
-            "system": "System:\n{system}\nUser:\n{user}\nAssistant:\n",
-            "no_system": "User:\n{user}\nAssistant:\n",
-        }
-
-    def format(self, sample, column_map=None):
-        if "system" in sample:
-            return self.template["system"].format(**sample)
-        else:
-            return self.template["no_system"].format(**sample)
+class DummyChatFormat:
+
+    B_SYS, E_SYS = "System:\n", "\n"
+    B_INST, E_INST = "User:\n", "\nAssistant:\n"
+    B_ASST, E_ASST = "", ""
+    system = f"{B_SYS}{{content}}{E_SYS}"
+    user = f"{B_INST}{{content}}{E_INST}"
+    assistant = f"{B_ASST}{{content}}{E_ASST}"
+
+    @classmethod
+    def format(
+        cls,
+        messages,
+    ):
+        formatted_dialogue = []
+        for message in messages:
+            content = ""
+            if message.role == "system":
```

Comment:
nit: could put system, user, assistant in a dictionary and just key on message.role